apps/api/src/controllers/api.controller.ts

```diff
@@ -3,7 +3,7 @@ import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
 import { ApiService } from '../services/api.service';
 
 @Controller('api')
-@ApiTags('account-service')
+@ApiTags('api')
```

Comment:
Question: I'm wondering how @Controller/@ApiTags works. Is it instantiating a controller each time the decorator is used, or will it just add to existing ones? More a question of how the underlying framework is executed, something I would like to understand but not a blocker here.

Reply:
did some research into this and it's basically just creating a label. doesn't see to change the structure in any way.